test/tvla_test.py

```diff
@@ -31,22 +31,22 @@ def ttest_significant(ttest_trace) -> bool:
 def test_general_nonleaking_project():
     project_path = TestDataPath('tvla_general/ci_opentitan_simple_kmac.cwp')
     tvla = TvlaCmd(Args(['--project-file', str(project_path),
-                        '--mode', 'kmac', 'run-tvla'])).run()
+                        '--mode', 'kmac', '--save-to-disk', 'run-tvla'])).run()
```

Comment:
I am not sure if we should add a specific argument to only save the ttest[-step] file. Because this argument here will also save the trace file to disk (a dense copy of the chipwhisperer traces). It might not be ideal for CI due to I/O limitations even though the trace sets here are small.

For non-CI applications, i.e., when large trace sets are analyzed, this will store several GBytes of data to disk which is slow. For this reason I originally only enabled `save_to_disk_trace` and `save_to_disk_trace` when number of steps is 1. That should maybe be reworked, can be done in another PR of course!

Reply:
I agree.
For now I've added `--save-to-disk-ttest` option and updated issue #72 to do the same for `--save-to-disk-leakage` and `--save-to-disk-trace`.